qiskit/circuit/gate.py

```diff
@@ -100,15 +100,15 @@ def control(
         """Return controlled version of gate. See :class:`.ControlledGate` for usage.
 
         Args:
-            num_ctrl_qubits: number of controls to add to gate (default=1)
+            num_ctrl_qubits: number of controls to add to gate (default:: 1)
```

Comment:
Doesn't a double colon trigger some kind of Sphinx magic? If yes we could also just do
```suggestion
            num_ctrl_qubits: number of controls to add to gate (defaults to 1)
```
but if it renders as desired, please ignore this comment 🙂